superset-frontend/src/views/CRUD/hooks.ts

```diff
@@ -598,6 +598,8 @@ export const copyQueryLink = (
 
 export const getDatabaseImages = () => SupersetText.DB_IMAGES;
 
+export const getConnectionAlert = () => SupersetText.ALERTS;
```

Comment:
can we change the superset_text key to say DB_CONNECTION_ALERTS. Then update this variable SupersetText.DB_CONNECTION_ALERTS